apps/omg_conformance/mix.exs

```diff
@@ -0,0 +1,36 @@
+defmodule OMG.Conformance.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :omg_conformance,
+      version: "#{String.trim(File.read!("../../VERSION"))}",
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.8",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      test_coverage: [tool: ExCoveralls]
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  defp elixirc_paths(:prod), do: ["lib"]
+  defp elixirc_paths(:dev), do: ["lib"]
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+
+  defp deps do
+    [
+      {:propcheck, "~> 1.1", only: [:test, :dev]},
```

Comment:
ok, so now since you don't compile it in dev anymore, you should be able to drop it from here too.